src/saga/transaction.ts

```diff
@@ -54,3 +54,31 @@ export const DISCORD_FETCH_MEMBERS_TRANSACTIONS: ITransaction[] = [
     ...DEFAULT_TRANSACTION_PROPERTY,
   },
 ];
+
+
+export const TWITTER_REFRESH_TRANSACTIONS: ITransaction[] = [
+  {
+    queue: Queue.TWITTER_BOT,
+    event: Event.TWITTER_BOT.EXTRACT.TWEETS,
+    order: 1,
+    ...DEFAULT_TRANSACTION_PROPERTY
+  },
+  {
+    queue: Queue.TWITTER_BOT,
+    event: Event.TWITTER_BOT.EXTRACT.PROFILES,
```

Comment:
`LIKES` is more important than the `PROFILES` for us, so I would suggest to move the `EXTRACT.PROFILES` below the `EXTRACT.LIKES`